### ISSUE TITLE
Batch mode statistical summary export fails "no valid measurements" despite there are data

### ISSUE THREAD

**Issue by jarogumulec** [ZELLMECHANIK-DRESDEN/ShapeOut#268]
### General information
- Win10 64
- ShapeOut 1.0.4. post1

### Steps to reproduce the problem
1. Open Shapeout
2. set and apply filter
3. batch - summary
4. select folder with data, select channel only, select output file, click Assemble

### Data files
not data related

### Expected behavior
summary is assembled

### Actual behavior
desipte there is data, (All measurements ##),  following error message occur, no export is created

```
Traceback (most recent call last):
File "shapeout\gui\batch.py". lie 193, in OnBatch
ValueError: No valied measireements with current selection!
```

**Comment by paulmueller** [comment]
I am labeling this as  a question, because I believe that it is not a bug: Please make sure that there are channel (not only reservoir) measurements in the selected folder. I cannot reproduce this issue. What happens if you choose "Channel and Reservoir" instead?

**Comment by jarogumulec** [comment]
When selecting folder(s) with both channel+reservoir measurements
- batch summary works when "Channel + reservoir" is selected
- batch summary does not work when either "Channel" or "Reservoir" is selected (same error for both of them)

**Comment by paulmueller** [comment]
Hmm... What kind of data are in the selected folder. .tdms-files or .rtdc files? Would it be possible for you to upload an example measurement somewhere?

**Comment by jarogumulec** [comment]
I am using .rtdc files only. I dont think its an issue of files as when I did such export on computers in MPL (while wisiting the lab, worked fine there), but I fail to do the same thing on my work PC in Czechia (so I first thought about an encoding-related problem as in https://github.com/ZELLMECHANIK-DRESDEN/ShapeOut/issues/267). 

an example of data here https://filesender.cesnet.cz/?s=download&token=2b6c81a3-eb3f-388c-9347-e17c53366281

**Comment by paulmueller** [comment]
Turns out that the chip region was stored as `Channel` instead of `channel` (lower-case) in the .rtdc files. Shape-Out did not expect such a thing.

**Comment by paulmueller** [comment]
should be fixed in 1.0.6 https://github.com/ZELLMECHANIK-DRESDEN/ShapeOut/releases/tag/1.0.6
